SPECS/pssh/pssh.spec

```
@@ -0,0 +1,132 @@
+%{?python_enable_dependency_generator}
+%global pypi_name pssh
+%global sum PSSH provides parallel version of OpenSSH and related tools
+%global desc PSSH provides parallel version of OpenSSH and related tools,\
+including pssh, pscp, prsync, pnuke and pslurp.\
+This project includes psshlib which can be used within custm applications.
+
+Name:           %{pypi_name}
+Version:        2.3.5
+Release:        1%{?dist}
+Summary:        %{sum}
```

After the updates `desc` seems unused and `sum` is only expanded in one place, so what do you think about this:
```suggestion

Name:           %{pypi_name}
Version:        2.3.5
Release:        1%{?dist}
Summary:        PSSH provides parallel version of OpenSSH and related tools
```

---

changes have been addressed